File: Data Structures & Algorithms/reverse-string/submission-1.js
class Solution {
    /**
     * @param {character[]} s
     * @return {void} Do not return anything, modify s in-place instead.
     */
    reverseString(s) {

        for(let i=0; i<s.length/2; i++) {
            let cur = s[i];

            s[i] = s[s.length - 1 -i ];
            s[s.length - 1 -i ] = cur
        };

        return s;
    }
}

File: Data Structures & Algorithms/move-zeroes/submission-1.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """

        writePointer = 0

        for readPointer in range(len(nums)):
            if nums[readPointer] is not 0:
                temp = nums[writePointer]
                nums[writePointer] = nums[readPointer]
                nums[readPointer] = temp
                writePointer += 1
        